mne/channels/channels.py

```diff
@@ -440,18 +440,36 @@ def inst_has(attr):
 class InterpolationMixin(object):
     """Mixin class for Raw, Evoked, Epochs
     """
-    def interpolate_bads_eeg(self):
-        """Interpolate bad channels
 
-        Operates in place.
+    def interpolate_bads(self, reset_bads=True):
```

Comment:
fine with me so you can have in the example:

```evoked_after = evoked_before.copy().interpolate_bads()```

don't forget to explicit in the docstring that the instance is modified inplace.